assets/constants.ts

```diff
@@ -6387,6 +6395,8 @@ export default {
   // Maximum allowed length of unique progress url ID.
   "MAX_PROGRESS_URL_ID_LENGTH": 6,
 
+  "PLATFORM_PARAMETER_ALLOWED_BROWSER_TYPES": [
+    "Chrome", "Edge", "Safari", "Firefox", "Others"],
```

Comment:
Why these changes?

Reply:
Well actually I did not change this, neither did I notice this change while pushing!?. This does need to be there, right?

Reply:
I see, please remove these changes as the browser_type filter for platform parameters have been deprecated.
Thanks

Reply:
Done